client/src/components/HomeScreen.tsx

```diff
@@ -72,9 +73,20 @@ const LoginWrapper = styled.div`
 `
 
 const LoginScreen: React.FC = () => {
+  const query = new URLSearchParams(useLocation().search)
+
   return (
     <LoginWrapper>
       <img height="50px" src="/arlo.png" alt="Arlo, by VotingWorks" />
+      {query.get('logged-out') && (
+        <Callout
+          icon="lock"
+          intent="warning"
+          style={{ margin: '20px 0 20px 0' }}
+        >
+          You have been logged out due to inactivity.
+        </Callout>
+      )}
```

Comment:
suggest doing a 

`location.replace('/')`

so that clicking the back button later doesn't make that notification show back up, which could be confusing.

Reply:
oh also that would work only if you change `?logged_out=1` to `#loggedout` as a fragment, so that `location.replace` doesn't actually do a page reload.

Reply:
updated f71da731